udagram-api-feed/docker-compose.yml

```diff
@@ -1,16 +1,16 @@
-version: '3.8'
-services:
-  postgres:
-    image: postgres:latest
-    container_name: udagram-db
-    ports:
-      - "5432:5432"
-    environment:
-      POSTGRES_DB: udagram-db
-      POSTGRES_USER: postgres
-      POSTGRES_PASSWORD: postgres
-    volumes:
-      - postgres-data:/var/lib/postgresql/data
-
-volumes:
+version: '3.8'
+services:
+  postgres:
+    image: postgres:latest
```

Comment:
use a fixed tag version, avoid using latest for constant image update. You could leverage alpine as well to reduce image size: https://hub.docker.com/_/postgres/tags?page=1&name=alpine